.github/workflows/agent-trigger.yml

```
@@ -7,19 +7,64 @@ on:
 jobs:
+  resolve-ids:
+    name: Resolve Board Column IDs
+    runs-on: ubuntu-latest
+    outputs:
+      project_id: ${{ steps.resolve.outputs.project_id }}
+      status_field_id: ${{ steps.resolve.outputs.status_field_id }}
+      status_development: ${{ steps.resolve.outputs.status_development }}
+    steps:
+      - name: Resolve column IDs by name
+        id: resolve
+        if: ${{ secrets.PROJECT_TOKEN != '' && vars.PROJECT_ID != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)